streaming/streaming.go

```diff
@@ -396,14 +390,40 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		fmt.Printf("flexTemplateRequest: %+v\n", req)
 		return fmt.Errorf("unable to launch template: %v", err)
 	}
+	reportOutput(conv, datastreamCfg, respDf, project)
+	return nil
+}
+
+func reportOutput(conv *internal.Conv, datastreamCfg DatastreamCfg, respDf *dataflowpb.LaunchFlexTemplateResponse, project string) {
```

Comment:
Suggest we create a more descriptive name. Report output seems very generic.

Reply:
Renamed to `printDataflowJob`